src/components/Card/Card.jsx

```diff
@@ -1,36 +1,79 @@
 import { Link } from "react-router-dom";
 import { useCart } from "../../context/cart-context";
 import { useWishlist } from "../../context/wishlist-context";
```

Comment:
You can import these in a single file and then export from there

Reply:
Alright, will keep it in mind.